charts/db-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
-kubeVersion: ">= 1.19-prerelease <= 1.22-prerelease"
+kubeVersion: ">= 1.19-prerelease <= 1.23-prerelease"
 appVersion: "1.4.0"
```

Comment:
@vladimir22 can you increase the app version to "1.5.0". Then I will approve and release the new version.

Reply:
@dabde [done](https://github.com/kloeckner-i/db-operator/pull/130/commits/3a3b28e5210d21c8084c103d576746294f00dd39), thank you very much :)